ExportAs/ExportAs.py

```diff
@@ -38,8 +38,11 @@ def __init__(self, scriptedPlugin):
     AbstractScriptedSubjectHierarchyPlugin.__init__(self, scriptedPlugin)
 
     self.exportAsAction = qt.QAction(f"Export as...", scriptedPlugin)
+    self.exportTfmAsAction = qt.QAction(f"Export Hardened as...", scriptedPlugin)
```

Comment:
Probably "Export transformed as..." would be more clear.

Reply:
Also we prefer whole names for the sake of clarity so please change `Tfm` to `Transformed` in all uses.

Reply:
Right!